### DISCUSSION TITLE
Auto-logout enforcement after a session timeout

### DISCUSSION THREAD

**Discussion by x-R4Z0R-x** [dfir-iris/iris-web#440] [Ideas]
Hi Guys,

I think it would be good if the user is auto-logged out after a configurable session timeout variable.

**Comment by x-R4Z0R-x** [comment]
Hi @whikernel,
Can you please check this? is it possible?

**Comment by whikernel** [comment]
Hi @x-R4Z0R-x 
It is by setting the `.env` with `IRIS_SESSION_TIMEOUT=xxx`, xxx being minutes. 

Cheers

**Comment by x-R4Z0R-x** [comment]
Thanks
